aspnetcore/host-and-deploy/iis/development-time-iis-support.md

```diff
@@ -5,71 +5,136 @@ description: Discover support for debugging ASP.NET Core apps when running behin
 manager: wpickett
 ms.author: riande
 ms.custom: mvc
-ms.date: 09/13/2017
+ms.date: 05/14/2018
 ms.prod: asp.net-core
 ms.technology: aspnet
 ms.topic: article
 uid: host-and-deploy/iis/development-time-iis-support
 ---
 # Development-time IIS support in Visual Studio for ASP.NET Core
 
-By [Sourabh Shirhatti](https://twitter.com/sshirhatti)
+By [Sourabh Shirhatti](https://twitter.com/sshirhatti) and [Luke Latham](https://github.com/guardrex)
 
 This article describes [Visual Studio](https://www.visualstudio.com/vs/) support for debugging ASP.NET Core apps running behind IIS on Windows Server. This topic walks through enabling this feature and setting up a project.
 
 ## Prerequisites
 
-[!INCLUDE [](~/includes/net-core-prereqs-windows.md)]
+[!INCLUDE[](~/includes/net-core-prereqs-windows.md)]
 
 ## Enable IIS
 
-Enable IIS. Navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen). Select the **Internet Information Services** checkbox.
+1. Navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen).
+1. Select the **Internet Information Services** check box.
 
-![Windows Features showing Internet Information Services checkbox checked as a black square (not a checkmark) indicating that some of the IIS features are enabled](development-time-iis-support/_static/enable_iis.png)
+![Windows Features showing Internet Information Services check box checked as a black square (not a checkmark) indicating that some of the IIS features are enabled](development-time-iis-support/_static/enable_iis.png)
 
 If the IIS installation requires a restart, restart the system.
```

Comment:
~~If the IIS installation requires a restart, restart the system.~~
The IIS installation may require a system restart.